index.cgi

```diff
@@ -25,7 +25,7 @@ cd "$CA_DIR"
 #1 Output
 sign() {
   local paramOutput=$1
-  unset cn ip ns days
+  unset cn ip ns days gp dn
```

Comment:
First of all, thank you for this PR!

DN is ***the*** field, please move it to the first position.